modules/node-tuning-hosted-cluster.adoc

```diff
@@ -0,0 +1,121 @@
+          profile:
+          - data: |
+              [main]
+              summary=Custom OpenShift profile
+              include=openshift-node
+              [sysctl]
+              vm.dirty_ratio="55"
+            name: tuned-1-profile
+          recommend:
+          - priority: 20
+            profile: tuned-1-profile
+
++
+[NOTE]
+====
+If you do not add any labels to an entry in the `spec.recommend` section of the tuned spec, node-pool-based matching is assumed, so the highest priority profile in the `spec.recommend` section is applied to nodes in the pool. Although you can achieve more fine-grained node-label-based matching by setting a label value in the tuned `.spec.recommend.match` section, node labels will not persist during an upgrade unless you set the node pool `.spec.management.upgradeType` value to `InPlace`.
+====
+
+. Create the config map in the management cluster:
+   
+   oc --kubeconfig="$MGMT_KUBECONFIG" create -f tuned-1.yaml
+
+. Reference the config map in the node pool's `spec.tunedConfig` field, either by editing a node pool or creating one. In this example, assume that you have only one node pool, `nodepool-1`, which contains 2 nodes.
+
+    apiVersion: hypershift.openshift.io/v1alpha1
+    kind: NodePool
+    metadata:
+      ...
+      name: nodepool-1
+      namespace: clusters
+    ...
+    spec:
+      ...
+      tunedConfig:
+      - name: tuned-1
+    status:
+    ...
+
++
+[NOTE]
+====
+You can reference the same config map in multiple node pools. In hosted control planes, the Node Tuning Operator appends a hash of the node pool name and namespace to the name of the tuned custom resources (tuneds) to distinguish them. Outside of this case, do not create multiple tuned profiles of the same name in different tuneds for the same hosted cluster.
+====
+
+.Verification
+
+Now that you have created the config map that contains a tuned manifest and referenced it in a node pool, the Node Tuning Operator syncs the tuned objects into the hosted cluster. You can verify which tuneds are defined and which profiles are set for each node.
+
+. List the tuned objects in the hosted cluster:
+
+    oc --kubeconfig="$HC_KUBECONFIG" get Tuneds -n openshift-cluster-node-tuning-operator
+
++
+.Example output
+[source,terminal]
+----
+    NAME       AGE
+    default    7m36s
+    rendered   7m36s
+    tuned-1    65s
+----
+   
+. List the profiles in the hosted cluster:
+
+    oc --kubeconfig="$HC_KUBECONFIG" get Profiles -n openshift-cluster-node-tuning-operator
+
++
+.Example output
+[source,terminal]
+----
+    NAME                           TUNED            APPLIED   DEGRADED   AGE
+    nodepool-1-worker-1            tuned-1-profile  True      False      7m43s
+    nodepool-1-worker-2            tuned-1-profile  True      False      7m14s
+----
+
++
+As you can see, both worker nodes in the node pool have the `tuned-1-profile` applied. If no custom profiles are created, the `openshift-node` profile is applied by default.
```

Comment:
The first sentence is unnecessary because the output shows the profile is applied. The second sentence should probably go in a [NOTE] block for emphasis. 

In addition, the `+` on the above line (107) needs to go directly under the output for it to nest correctly under the step. 
Example:
```
.Example output
[source,terminal]
----
    NAME                           TUNED            APPLIED   DEGRADED   AGE
    nodepool-1-worker-1            tuned-1-profile  True      False      7m43s
    nodepool-1-worker-2            tuned-1-profile  True      False      7m14s
----
+
[NOTE]
====
If no custom profiles are created, the `openshift-node` profile is applied by default.
===
```